pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -1604,7 +1587,7 @@ func (cc *XController) Run(stopCh chan struct{}) {
 	go wait.Until(cc.PreemptQueueJobs, 60*time.Second, stopCh)
 
 	// This thread is used as a heartbeat to calculate runtime spec in the status
-	go wait.Until(cc.UpdateQueueJobs, 5*time.Second, stopCh)
+	//go wait.Until(cc.UpdateQueueJobs, 5*time.Second, stopCh)
```

Comment:
commenting this thread out will cause AWs not to be enqueued, I am curios why is this not needed now.

Reply:
Will removed the comment in the final commit. I just needed to remove another point where the AW is updated.

Do you have any ideas as to why this function is needed? What are the scenarios that this would actually do something useful?